client/src/components/Items.js

```diff
@@ -49,148 +45,131 @@ const Items = ({ windowSize }) => {
         cols={windowSize.width > 766 ? 3 : 2}
         gap={window.innerWidth > 339 ? 16 : 8}
       >
-        {images.map(({ id, img }) => (
-          <div key={id} className="item-list-wrapper">
-            {/* substitute heart icon for MoreVertIcon when closet does not belong to the person who is logged in */}
-            {/* <button>
+        {userData?.closet?.map(
```

Comment:
I think the error is coming from here (if you don't have items in the closet to map).

I was able to get rid of the error by wrapping this in a ternary operator like:

```
{userData?.closet ? [your whole map function here] : ""}
```

That ensures that if there isn't a closet to render, at least an empty string is returned, rather than undefined.

Reply:
Yes, that fixed it. Thank you!